docs/pages/application-access/guides/azure.mdx

```diff
@@ -0,0 +1,554 @@
+within your organization. This group must also include your user.
+
+Here is an example:
+
+```diff
+  attributes_to_roles:
+    - name: "groups" 
+      value: "my-group" 
+      roles:
+        - access
++       - azure-cli-access
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f saml.yaml
+```
+</TabItem>
+<TabItem label="OIDC">
+
+Retrieve your `oidc`  configuration resource:
+
+```code
+$ tctl get oidc/myoidc > oidc.yaml
+```
+
+Edit `oidc.yaml`, adding `azure-cli-access` to the `claims_to_roles` section.
+The claim you will map to this role will depend on how you have designed your
+organization's RBAC, but it should be the smallest group possible within your
+organization. This group must also include your user.
+
+Here is an example:
+
+```diff
+  claims_to_roles:
+    - name: "groups" 
+      value: "my-group" 
+      roles:
+        - access
++       - azure-cli-access
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f saml.yaml
+```
+</TabItem>
+</Tabs>
+
+Log out of your Teleport cluster and log in again to assume the new role.
+
+<Details title="Configuring Azure identities per user">
+
+The instructions above show you how to define a role with access to specific
+Azure identities, which means that Teleport users who assume this role can use
+those (and only those) identites to execute commands via the Azure CLI. Another
+approach is to define a user's permitted Azure identities by configuring each
+user separately.
+
+To do so, modify the `azure-cli-access` role you defined above to include the
+following:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: azure-cli-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    azure_identities:
+      - '{{internal.azure_identities}}'
+```
+
+In this case, when a user authenticates to the Azure CLI via Teleport, the
+Teleport Auth Service populates the `{{internal.azure_identities}}` template
+variable with any Azure identities you have assigned to the user.
+
+To assign Azure identities to a user, run a `tctl users update` command similar
+to the following:
+
+```code
+$ TELEPORT_USER=myuser;
+$ ID1=one_id;
+$ ID2=another_id;
+$ SUBSCRIPTION_ID=00000000-0000-0000-0000-000000000000;
+$ RESOURCE_GROUP="my-resource-group";
+$ tctl users update ${TELEPORT_USER?} --set-azure-identities \
+ `/subscriptions/${SUBSCRIPTION_ID?}/resourceGroups/${MY_RESOURCE_GROUP?}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/${ID1?},\
+ `/subscriptions/${SUBSCRIPTION_ID?}/resourceGroups/${MY_RESOURCE_GROUP?}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/${ID2?}
+```
+
+This command uses the `--set-azure-identities` flag to add Azure identities to a
+user. The value of this flag is a comma-separated list of Azure identity URIs.
+
+You must attach all identities you would like users to access to the Azure VM
+running the Application Service so the Application Service can make requests to
+the Azure API using those identities.
```

Comment:
This seems technically correct and all that, but it's a mouthful and took me a couple passes to parse. Not sure how to best rephrase, maybe...

```suggestion
You must attach all identities you would like users to access to the Azure VM
running the Application Service. This lets the Application Service make requests to
the Azure API using those identities.
```

Something like that, splitting up the single sentence?